content/blog/2019-01-06-babette.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: babette
+title: Call BEAST2 from R
+package_version: 2.0
+authors:
+  - Richèl J.C. Bilderbeek
+date: 2019-01-06
```

Comment:
```suggestion
date: 2020-01-28
```

set publication date (happy anniversary!)

Reply:
Updated again to today.